### PR TITLE
added epnNetwork datatype

### PR DESCRIPTION
**Pull request by ywkim312**
@navarroc @longshuicy @nunezco2 Is there any other dataType to be added, like pipeline network or others? Also, is waternetwork should be 'waternetwork' or 'waterNetwork'?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+### Added
+- EPN network data type [#24](https://github.com/IN-CORE/incore-data-release/issues/24)
+
```
Comment (nunezco2) [inline CHANGELOG.md:11]: Add waterNetwork data type

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
...
 ## [3.5.0] - 2022-07-27
 ### Added
 - Water facility network and Electric power network datasets for MMSA Shelby [#19](https://github.com/IN-CORE/incore-data-release/issues/19)
```
Comment (jonglee1) [inline CHANGELOG.md:14]: * Potable water network instead of "water facility network"
* no need "Shelby". just MMSA

File: CHANGELOG.md (hunk 1 above)
Comment (ywkim312) [reply CHANGELOG.md:11]: waterNetwork was already in there

File: CHANGELOG.md (hunk 2 above)
Comment (ywkim312) [reply CHANGELOG.md]: modified

**Review by jonglee1 — APPROVED** [review]

**Review by nunezco2 — APPROVED** [review]

**Pull request merged**